packages/editor/src/pages/Auth/index.tsx

```diff
@@ -83,18 +88,9 @@ class AuthPage extends React.Component<IProps, IState> {
         'and retrieve a new sign-in URL to open in a new page.';
     }
 
-    let publicKey: NodeRSA;
-    try {
-      publicKey = new NodeRSA(atob(base64Key));
-    } catch (e) {
-      error =
-        `The "key" parameter in the URL appears to be incomplete. ` +
-        `Please go back to the sign-in dialog in the code editor, and be sure to copy the full URL.`;
-    }
-
     this.state = {
       isIE,
```

Comment:
isIE - isn't this implied? Why else would we be in this codepath?

Reply:
There's a chance someone opened this page in IE (say, because it's their default browser).  So I still need to handle the unhappy path of them ending up in that state.